### PR TITLE
Codeception 5 Support

### PR DESCRIPTION
**Pull request by TavoNiievez**
Co-authored-by: Luke English <luke.english@ec2i.biz>

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,19 +16,20 @@
...
-        "codeception/codeception": "^4.0"
+        "php": "^8.0",
+        "codeception/codeception": "^5.0.0-alpha2",
+        "codeception/lib-innerbrowser": "^3.0"
     },
     "require-dev": {
         "yiisoft/yii2": "dev-master",
         "yiisoft/yii2-app-advanced": "dev-master",
-        "codeception/verify": "<2",
+        "phpunit/phpunit": "dev-master as 9.5.99",
```
Comment (Naktibalda) [inline composer.json:28]: Please remove constraint of `phpunit/phpunit` unless you really have dependency on PHPUnit 10.

**Comment by developedsoftware** [comment]
Can this be merged?

**Comment by samdark** [comment]
@developedsoftware any idea why it fails tests?

**Comment by developedsoftware** [comment]
If you merge [this](https://github.com/Codeception/module-yii2/pull/69) pull request into branch codecept5 then this should pass also. 

It had a reference to swiftmailer still in the yii2 codeception module.

**Comment by developedsoftware** [comment]
Will 1.1.6 be released when this is merged into master?

**Comment by samdark** [comment]
Thanks!

**Comment by samdark** [comment]
Done.

**Pull request merged**